todo_app/templates/index.html

```diff
@@ -2,17 +2,51 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+<div class="jumbotron">
+  <h1 class="display-4">To-Do App</h1>
+  <p class="lead">Just another to-do app.</p>
+</div>
+<div class="row">
+  <div class="container mb-3">
+    <form action="/todos" method="post">
+      <div class="mb-3">
+        <label for="title" class="form-label">New Todo</label>
+        <input type="text" id="title" name="title" class="form-control" />
```

Comment:
We could add a `required` tag to this input, which will guide our browser to help us avoid accidentally submitting an empty todo